src/sqlfluff/dialects/dialect_hive.py

```diff
@@ -448,25 +448,12 @@ class DropStatementSegment(BaseSegment):
     match_grammar = StartsWith("DROP")
     parse_grammar = OneOf(
         Ref("DropDatabaseStatementSegment"),
+        Ref("DropSchemaStatementSegment"),
         Ref("DropTableStatementSegment"),
```

Comment:
Good example here of where other dialects are already splitting up the `DropStatementSegment` from ANSI so good to make ANSI consistent 👍

Reply:
ack